velox/exec/Driver.h

```diff
@@ -244,6 +244,36 @@ struct DriverCtx {
   std::optional<common::SpillConfig> makeSpillConfig(int32_t operatorId) const;
 };
 
+/// Enum for operator methods we want to track.
+enum class OpMethod {
+  kNone,
+  kIsBlocked,
+  kNeedsInput,
+  kGetOutput,
+  kAddInput,
+  kNoMoreInput
+};
+
+/// Structure holds the information about the current operator call the driver
+/// is in. Can be used to detect deadlocks and otherwise blocked calls.
+/// If timeStartMs is zero, then we aren't in an operator call.
+struct OpCallStatus {
+  std::atomic_size_t timeStartMs{0};
+  std::atomic_int32_t opId{0};
```

Comment:
Perhaps, add a comment to clarify what this ID is?

Reply:
@mbasmanova 
Sure!
Should we add a comment for every member of this struct?

Reply:
That would be nice. Thanks.